packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -166,13 +166,12 @@ export abstract class TelemetryLogger implements ITelemetryLogger {
      * @param error - optional error object to log
      */
     public sendErrorEvent(event: ITelemetryErrorEvent, error?: any) {
+        // ensure the error field has some value even if no error object is given (will generate stack too)
+        const errorForLogging = error ?? event.error ?? event.eventName;
```

Comment:
I personally would prefer to collect stack without this change - I do not see much value in replicating eventName as an error

Reply:
I don't mind that part so much, I can see it both ways. Let's give it some time and see how it feels in Kusto before trying to make a case to revert that.